metadata.rb

```diff
@@ -18,6 +19,7 @@
 depends          'osl-mysql'
 depends          'osl-nginx'
 depends          'osl-nodejs'
+depends          'percona'
```

Comment:
You don't need to include this directly here as it will be pulled in via osl-mysql.